evaporate.js

```diff
@@ -14,7 +14,7 @@
 
 /***************************************************************************************************
  *                                                                                                 *
- *  version 2.0                                                                                    *
+ *  version 2.1                                                                                    *
```

Comment:
What about using 2.0.1? Semver says bug fixes should increment the patch version.